src/circuit/lookup.rs

```diff
@@ -20,13 +31,12 @@ fn synth<'a, E: Engine, I>(
     assert_eq!(assignment.len(), 1 << window_size);
 
     for (i, constant) in constants.into_iter().enumerate() {
-        let mut cur = assignment[i];
-        cur.negate();
-        cur.add_assign(constant);
-        assignment[i] = cur;
+        assignment[i].add_assign(constant);
+        let cur = assignment[i];
+        // subtract a_i from a_j when the set bits of j contain those of i
```

Comment:
Make this a sentence.

Reply:
Capitilized and added "."  if that's what you meant